admin/templates/register.html

```diff
@@ -0,0 +1,73 @@
+{% load render_bundle from webpack_loader %}
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>OSFAdmin | Log in</title>
+    <!-- Tell the browser to be responsive to screen width -->
+    <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
+    <!-- Bootstrap 3.3.4 -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet" type="text/css" />
+    <!-- Font Awesome Icons -->
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+    <!-- Theme style -->
+    <link href="{% static 'css/AdminLTE.min.css' %}" rel="stylesheet" type="text/css" />
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+ <body class="register-page">
+  <div class="register-box">
+    <div class="register-logo">
+      <b>OSF</b>Admin
+    </div>
+    <div class="register-box-body">
+      <p class="login-box-msg">Register a new account</p>
+      <form action="" method="post">
+      {% csrf_token %}
+       <div class="form-group">
+          <label for="email">Email:</label>
+          <p>{{ form.email }}</p>
+          {% if form.email.errors %}<span class="text-danger"> {{ form.email.errors }}</span>{% endif %}
+      </div>
+      <div class="form-group">
+          <label for="first_name">First name:</label>
+          <p>{{ form.first_name }}</p>
+          {% if form.first_name.errors %}<span class="text-danger"> {{ form.first_name.errors }}</span>{% endif %}
+      </div>
+      <div class="form-group">
+          <label for="last_name">Last name:</label>
+          <p>{{ form.last_name }}</p>
+          {% if form.last_name.errors %}<span class="text-danger"> {{ form.last_name.errors }}</span>{% endif %}
+      </div>
+       <div class="form-group">
```

Comment:
IIRC we discussed this. You probably just haven't gotten around to changing it yet but my version looks like this. You want to have both password fields there so people don't accidentally mistype their password and we don't validate it against the other entry.

```html
          <div class="form-group">
              <label for="password1">Password:</label>
              <p>{{ form.password1 }}</p>
              {% if form.password1.errors %}<span class="text-danger"> {{ form.password1.errors }}</span>{% endif %}
          </div>
       <div class="form-group">
          <label for="password2">Confirm Password:</label>
          <p>{{ form.password2 }}</p>
          {% if form.password2.errors %}<span class="text-danger"> {{ form.password2.errors }}</span>{% endif %}
      </div>
```